pyproject.toml

```diff
@@ -3,7 +3,7 @@ requires = [
   "setuptools>=62.1",
   "setuptools_scm[toml]>=8.0.1",
   "wheel",
-  "extension-helpers",
+  "extension-helpers>=1.3.0",
```

Comment:
I'd recommend adding <2 as well as in general we recommend pinning to a major version